packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -99,6 +107,9 @@ export function getDeserializePrivateFunction(
       `}`
     );
   } else if (returnType.type === "void") {
+    if (statements.length === 0) {
+      statements.push(`result;`);
+    }
```

Comment:
what is this trying to resolve ?

Reply:
https://github.com/Azure/autorest.typescript/pull/1931/files#diff-3d6099d37e97ef4ac1cb7f3b96c64afb5f22edfccbeda5560dc70e0474bf7e62R34-R39